src/d3-ext/profile.js

```diff
@@ -58,13 +62,13 @@ ngeo.profile = function() {
    * @type {function(Object)}
    * Hover callback function.
    */
-  var hoverCallback = goog.nullFunction;
+  var hoverCallback = options.hoverCallback || goog.nullFunction;
```

Comment:
@elemoine, I think rules should be flexible enough so that they do not hinder developer creativity. Nevertheless, I updated the PR with your requirements.